client/dist/styles.css

```diff
@@ -1,33 +1,32 @@
-@font-face {
+/* @font-face {
```

Comment:
Hey, your CSS looks pretty good to me! You told me you would break this out into CSS modules though, so please do that before shipping :)